content/md/en/docs/reference/substrate-connect.md

```diff
@@ -0,0 +1,200 @@
+
+*Simple usage (Using an integrated, suported chain)*
+```js
+import { ScProvider, WellKnownChain } from '@polkadot/rpc-provider/substrate-connect';
+import { ApiPromise } from '@polkadot/api';
+// Create the provider for a known chain
+const provider = new ScProvider(WellKnownChain.westend2);
+// Stablish the connection (and catch possible errors)
+await provider.connect()
+// Create the PolkadotJS api instance
+const api = await ApiPromise.create({ provider });
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+console.log(lastHeader.hash);
+});
+await api.disconnect();
+```
+
+*Simple usage (custom chain/chainspec)*
+```js
+import { ScProvider } from '@polkadot/rpc-provider/substrate-connect';
+import { ApiPromise } from '@polkadot/api';
+import jsonCustomSpec from './jsonCustomSpec.json';
+// Create the provider for the custom chain
+const customSpec = JSON.stringify(jsonCustomSpec);
+const provider = new ScProvider(customSpec);
+// Stablish the connection (and catch possible errors)
+await provider.connect()
+// Create the PolkadotJS api instance
+const api = await ApiPromise.create({ provider });
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+console.log(lastHeader.hash);
+});
+await api.disconnect();
+```
+
+*Parachains usage*
+
+```js
+import { ScProvider, WellKnownChain } from '@polkadot/rpc-provider/substrate-connect';
+import { ApiPromise } from '@polkadot/api';
+import jsonParachainSpec from './jsonParachainSpec.json';
+// Create the provider for the relay chain
+const relayProvider = new ScProvider(WellKnownChain.westend2);
+// Create the provider for the parachain. Notice that
+// we must pass the provider of the relay chain as the
+// second argument
+const parachainSpec = JSON.stringify(jsonParachainSpec);
+const provider = new ScProvider(parachainSpec, relayProvider);
+// Stablish the connection (and catch possible errors)
+await provider.connect()
+// Create the PolkadotJS api instance
+const api = await ApiPromise.create({ provider });
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+console.log(lastHeader.hash);
+});
+await api.disconnect();
+```
+
+## Advanced Usage
+
+At the section above, `@polkadot/rpc-provider/substrate-connect` was used, because this provider makes if straightforward to create applications that interact with the chain using the Polkadot-JS API. For more advanced application development that doesn't depend on the Polkadot-JS API, you can install and use @substrate-connect. For example, if you are building your own application library or programming interfaces, you should install the Substrate Connect dependencies by running the following command:
+
+
+```bash
+yarn add @substrate/connect
+```
+or
+```bash
+npm i @substrate/connect
+```
+
+*Connecting to a `WellKnownChain`*
```

Comment:
```suggestion
### Connect to a well-known chain
```

Reply:
`WellKnownChain` is a wording/variable that is used inside the codebase and we should probably keep it that way

```suggestion
*Connect to a `WellKnownChain`*
```

Reply:
I'm suggesting we make it a heading and e don't use the back ticks in headings. Even though `WellKnownChain` is a direct reference to code, I don't think it is necessary to call attention to that in a heading. It doesn't improve the readability for someone scanning documentation and it is easier on the human eye to reduce the use of capital letters. When it's not in a heading, though, we can certainly use `WellKnownChain`.